spec/iex/resources/chart_spec.rb

```diff
@@ -72,4 +77,14 @@
       expect { subject }.to raise_error IEX::Errors::SymbolNotFoundError, 'Symbol INVALID Not Found'
     end
   end
+
+  context 'with client error', vcr: { cassette_name: 'chart/badRequest' } do
```

Comment:
Lets rename the `badRequest` file here to `bad_option` to match the name of the bad option.